src/libPMacc/include/cuSTL/cursor/SafeCursor.hpp

```diff
@@ -128,7 +128,9 @@ class SafeCursor : public Cursor
     HDINLINE void checkValidity() const
     {
         if(!this->enabled) return;
+#ifndef _MSC_VER
```

Comment:
that looks a bit odd...

also it;s interesting, since `#pragma unroll` is documented in the CUDA C reference guide as (the only?) way to unroll loops on the device...